website/docs/user_guide/personal-api-token.mdx

```diff
@@ -0,0 +1,42 @@
+---
+id: personal-api-token
+title: How to create Personal API Tokens
+---
+
+:::info availability
+
+Personal API tokens are planned to be released in **Unleash 4.17**.
+
+:::
+
+Personal API tokens are a tool to enable a user to use the Admin API as themselves with their own set of permissions, rather than using an admin token. See [_how to use the Admin API_](../advanced/api_access) for more information.
```

Comment:
I suggest linking to the reference docs. I'm also hesitant about linking to the api-access doc. I think it's a bit outdated. Do you think this would be sufficient?

```suggestion
[Personal API tokens](../reference/api-tokens-and-client-keys.mdx#personal-access-tokens) enable a user to use the Unleash admin API as themselves instead of as an global "admin" user.
```

Reply:
Yeah, I think that looks good, this is probably me getting too excited about adding links